myapp/app/models/task.rb

```diff
@@ -4,4 +4,8 @@ class Task < ApplicationRecord
     validates :name, presence: true
     validates :status, presence: true, inclusion: { in: STATUSES }
     validates :priority, presence: true, inclusion: { in: PRIORITY }
+
+    def self.ransackable_attributes(auth_object= nil)
+        ["name", "description", "status", "priority", "duedate"]
+    end    
```

Comment:
Indent of code guideline in Rakuma is two spaces.
so please remove spaces 🙏 

```suggestion
    def self.ransackable_attributes(auth_object= nil)
      ["name", "description", "status", "priority", "duedate"]
    end    
```

Reply:
Done.